### DISCUSSION TITLE
Error as "Unsupported data type" in getting_started_with_agents/step7_assistant.py  from Github repo

### DISCUSSION THREAD

**Discussion by rakshahulle** [microsoft/semantic-kernel#9544] [Q&A] [answered]
I am trying Python samples from getting_started_with_agents/step7_assistant.py. I have updated some code. I am getting an error as "Unsupported data type".
[step7_assistant.txt](https://github.com/user-attachments/files/17634385/step7_assistant.txt)

**Comment by moonbox3** [comment]
@rakshahulle, similar to the error that you resolved yesterday, can you please check your deployment name, your endpoint, and your API version? We're using the Azure OpenAI Assistant V2 API and it requires a certain model/deployment/API Version.

For the API version, can you configure your `AZURE_OPENAI_API_VERSION` as `2024-10-21` which is the latest GA API VERSION? https://learn.microsoft.com/en-us/azure/ai-services/openai/api-version-deprecation

Which model/deployment are you using as part of Azure OpenAI?

**Comment by rakshahulle** [comment]
I am using GPT 4o mini.
deployment_name="gpt-4o-mini",
endpoint="https://openaisk123.openai.azure.com/openai/deplo*yments/gpt*
-4o-mini/chat/completions?api-version=2024-08-01-preview",

**Reply by moonbox3** [reply, marked as answer]
Thanks for your response. The `endpoint` should be configured like:

`AZURE_OPENAI_ENDPOINT="https://<your_resource>.openai.azure.com/"`

The proper url will be built up using the `endpoint` along with your `deployment_name` and the `api-version`.

**Comment by rakshahulle** [comment]
Thank you so much. It worked
